resources/js/Pages/Home.vue

```diff
@@ -4,22 +4,30 @@
     <h1>Mismatch Finder</h1>
     <p>This page is coming soon!</p>
     <auth-widget v-bind:user="user" />
+    <Button type="progressive" variant="primary" @click.native="console">Test Wikit Component</Button>
```

Comment:
This looks great! What you mean by "adding routing to the button"?

To me, it seems ready: the button looks like we expect it too, and behaves like we expect it too.

Reply:
I wanted to make the button go to the `/store` route. But didn't manage so I removed it. But yes I agree that we can deal with this when we need to.

Reply:
Ah, in this case we want to use a link that looks like a button, rather than a button.

Reply:
Ah, much better then. :)